<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grive</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../mnt/safer/home/jpuchky/Projects/grive/icon/32x32/32x32.png</normaloff>../../../../../../mnt/safer/home/jpuchky/Projects/grive/icon/32x32/32x32.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="m_dir">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Preview</string>
         </attribute>
         <widget class="QGraphicsView" name="preview">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>381</width>
            <height>481</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Details</string>
         </attribute>
         <widget class="QCalendarWidget" name="calendarWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>294</y>
            <width>381</width>
            <height>181</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="m_title">
          <property name="geometry">
           <rect>
            <x>82</x>
            <y>10</y>
            <width>291</width>
            <height>25</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="lbl_m_title">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>12</y>
            <width>56</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Title</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="m_mime_type">
          <property name="geometry">
           <rect>
            <x>82</x>
            <y>40</y>
            <width>291</width>
            <height>25</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="lbl_m_mime_type">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>45</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Mime-type</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="usageBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="m_statusbar"/>
  <action name="m_action_exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../mnt/safer/home/jpuchky/Projects/grive/icon/16x16/16x16.png</normaloff>../../../../../../mnt/safer/home/jpuchky/Projects/grive/icon/16x16/16x16.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About grive</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionTopic">
   <property name="text">
    <string>Topic</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="text">
    <string>Console</string>
   </property>
   <property name="toolTip">
    <string>Open console window</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_action_exit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
